python/tests/ecl_tests/test_grid_generator.py

```diff
@@ -113,7 +116,8 @@ def test_extract_grid_slice_spec(self):
     def assertSubgrid(self, grid, subgrid, ijk_bound):
         sijk_space = prod(*[range(d) for d in subgrid.getDims()[:-1:]])
         for sijk in sijk_space:
-            gijk = tuple([a+b for a, b in zip(sijk, zip(*ijk_bound)[0])])
+            gijk = tuple([a + b for a, b in
+                         zip(sijk, list(zip(*ijk_bound))[0])])
```

Comment:
Here you actually don't need the `[...]`, you could write `tuple(a + b for a, b in ...)` but just leave it as it is.